packages/io/src/analyze/comlinkWorkerPool.js

```diff
@@ -0,0 +1,31 @@
+import { WorkerPool } from 'itk-wasm';
+import * as Comlink from 'comlink';
+
+export function comlinkWorkerPool(numberOfWorkers, makeWorker, operationName) {
```

Comment:
This is generally useful -- could it be added to `itk-wasm`?

Reply:
Can do.  Where to put?